packages/xstate-solid/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export { useActor } from './useActor';
+export { createService } from './createService';
+export { useMachine } from './useMachine';
+export type { UseMachineReturn } from './types';
```

Comment:
I would drop this public export for the time being to be consistent between packages - we can revisit this separately:
```suggestion
```